09-intermediate-rails/README.md

```diff
@@ -1,16 +1,19 @@
 # Rails Intermediate Index
 
-## Part 1 - Authentication and Intermediate Rails Topics
+## More Ruby on Rails
 1. [Flash](./flash.md)
 1. [Session](./session.md)
 1. [Testing Session](./testing-session.md)
 1. [Controller Filters](./controller-filters.md)
 1. [View helpers](./view-helpers.md)
-1. [Intro to OAuth](./intro-to-oauth.md)
-1. [OAuth in Rails](./oauth.md)
+1. [Advanced Validations](./advanced-validations.md)
+1. [Testing Controllers: Revisited](testing-controllers-revisited.md)
+
```

Comment:
Link on line 10 is broken

Reply:
Same as above-- this file was moved as part of this PR so as far as I can tell, this works?

Reply:
Right you are.